torch/lib/THD/base/ChannelUtils.cpp

```diff
@@ -181,28 +181,29 @@ int connect(const std::string& address, port_type port, bool wait, int timeout)
       }
 
       socklen_t err_len = sizeof(errno);
-      SYSCHECK(getsockopt(socket, SOL_SOCKET, SO_ERROR, &errno, &err_len));
+      ::getsockopt(socket, SOL_SOCKET, SO_ERROR, &errno, &err_len);
```

Comment:
Hmm, the only place errno can be set is `::poll` but we check return code, so I guess this line always start with `errno = 0` or I miss something?